lb_content_resolver/unresolved_recording.py

```diff
@@ -42,13 +42,33 @@ def add(self, recording_mbids):
             of times it has been unresolved.
         """
 
-        query = """INSERT INTO unresolved_recording (recording_mbid, last_updated, lookup_count)
-                        VALUES (?, ?, 1)
-         ON CONFLICT DO UPDATE SET lookup_count = EXCLUDED.lookup_count + 1"""
+        recording_mbids = list(set(recording_mbids))
+
+        placeholders = ",".join(("?", ) * len(recording_mbids))
+        existing = {
+                row[0]:True for row in db.execute_sql(
+                """SELECT recording_mbid
+                     FROM unresolved_recording
+                    WHERE recording_mbid IN (%s)""" %
+                placeholders, tuple(recording_mbids)).fetchall()
+        }
 
         with db.atomic() as transaction:
             for mbid in recording_mbids:
-                db.execute_sql(query, (mbid, datetime.datetime.now()))
+                if mbid in existing:
+                    db.execute_sql("""UPDATE unresolved_recording
+                                         SET lookup_count = lookup_count + 1,
+                                             last_updated = ?
+                                       WHERE recording_mbid = ?""", (datetime.datetime.now(), mbid))
+                else:
+                    db.execute_sql("""INSERT INTO unresolved_recording (recording_mbid, last_updated, lookup_count)
+                                           VALUES (?, ?, ?)""", (mbid, datetime.datetime.now(), 1))
+
```

Comment:
Since `datetime.datetime.now()` in both branches and in a loop, and better have one time for the whole transaction, I would rather move it above:

```python
now = datetime.datetime.now()
for mbid in recording_mbids:
```

And ofc use `now` instead.

Reply:
OK, makes sense. done.